src/components/RandomAppList.js

```diff
@@ -40,15 +40,22 @@ const appList = [
   },
 ]
 
-const RandomAppList = () => {
-  const list = appList.map((item) => {
-    item.randomNumber = Math.random()
-    return item
-  })
-  list.sort((a, b) => a.randomNumber - b.randomNumber)
+const RandomAppList = () =>
+const [setAppRandomList, appRandomList] = useState({ dapps: []});
```

Comment:
The set state function should be the second item in the array.

Reply:
I also don't think we need an object here (`useState({ dapps: []})`), I think we can just use an array `useState([])`